core-java-sample/src/main/java/org/spine3/sample/Application.java

```diff
@@ -29,6 +29,7 @@
 import org.spine3.base.UserId;
 import org.spine3.sample.order.OrderId;
 import org.spine3.sample.order.OrderRepository;
+import org.spine3.sample.saga.SagaRepositorySample;
```

Comment:
Please have package `procman` for everything that was `saga` or `process`.
Please rename Saga to ProcessManager everywhere too.